cinema/models.py

```diff
@@ -35,12 +39,19 @@ def full_name(self):
         return f"{self.first_name} {self.last_name}"
 
 
+def movie_image_path(movie: "Movie", filename: str) -> pathlib.Path:
+    ext = pathlib.Path(filename).suffix
+    filename = f"{slugify(movie.title)}-{uuid.uuid4()}{ext}"
+    return pathlib.Path("upload/movies/") / pathlib.Path(filename)
```

Comment:
```suggestion
    return pathlib.Path("upload") / "movies" / filename
```
Can you try like this?